workspace/errors.go

```diff
@@ -0,0 +1,19 @@
+package workspace
+
+import "errors"
+
+// ErrNotInWorkspace signals that the target directory is outside the configured workspace.
+var ErrNotInWorkspace = errors.New("not in workspace")
+
+// ErrNotExist signals that the target directory could not be located.
+var ErrNotExist = errors.New("not found")
+
+// IsNotInWorkspace checks if this is an ErrNotInWorkspace error.
+func IsNotInWorkspace(err error) bool {
+	return err == ErrNotInWorkspace
```

Comment:
I haven't seen this done very much in go. Is this a different way of doing something like `err.(*ErrNotInWorkspace)`? I need to brush up again in error handling 😞